test/solve-overview.spec.js

```diff
@@ -63,33 +64,6 @@ describe('Solve', () => {
         expect(this.wrapper.text()).to.contain('read');
       });
 
-      it('should display author when syndicated content is unchecked', async () => {
-        await this.wrapper.vm.getData();
-        const first = this.wrapper.vm.content[0];
-        first.field_tl_ = first.field_tl_.replace(this.wrapper.vm.tokens.syndicated, '');
-        // Check featured post.
-        expect(this.wrapper.text()).to.not.contain('Roger Avalos & HEB');
-        expect(this.wrapper.text()).to.contain('Mike Rustyellow');
-      });
-
-      it('should display author when syndicated content is null', async () => {
-        await this.wrapper.vm.getData();
-        const first = this.wrapper.vm.content[0];
-        first.field_tl_ = first.field_tl_.replace(this.wrapper.vm.tokens.syndicated, '');
-        // Check featured post.
-        expect(this.wrapper.text()).to.not.contain('Roger Avalos & HEB');
-        expect(this.wrapper.text()).to.contain('Mike Rustyellow');
-      });
```

Comment:
I don't think we want to drop both of these cases. I think we can just have a single case that states "it should display author". We no longer care about it being syndicated or not. We just want to verify that the author is showing up.